web/js/tasks.js

```diff
@@ -1163,32 +1163,35 @@ Ext.onReady(function(){
     });
 
     // Expand/collapse all Panel
+    var expandButton = new Ext.Button({
+        text:'Expand all',
+        flex: 2,
+        handler: function() {
+            var panels = tasksScrollArea.items;
+            for(var i=0; i<panels.getCount(); i++) {
+                panels.get(i).expand();
+            }
+            cookieProvider.set('tasksCollapsed', false);
+        }
+    });
+    var collapseButton = new Ext.Button({
+        text:'Collapse all',
+        flex: 2,
+        handler: function() {
+            var panels = tasksScrollArea.items;
+            for(var i=0; i<panels.getCount(); i++) {
+                panels.get(i).collapse();
+            }
+            cookieProvider.set('tasksCollapsed', true);
+        }
+    });
+
     var expandCollapseAllPanel = new Ext.Panel({
-        defaults: {
-            width: '100%',
-        },
-        items: [
-            new Ext.Button({
-                text:'Expand all',
-                handler: function() {
-                    var panels = tasksScrollArea.items;
-                    for(var i=0; i<panels.getCount(); i++) {
-                        panels.get(i).expand();
-                    }
-                    cookieProvider.set('tasksCollapsed', false);
-                }
-            }),
-            new Ext.Button({
-                text:'Collapse all',
-                handler: function() {
-                    var panels = tasksScrollArea.items;
-                    for(var i=0; i<panels.getCount(); i++) {
-                        panels.get(i).collapse();
-                    }
-                    cookieProvider.set('tasksCollapsed', true);
-                }
-            })
-        ]
+        //defaults: {
+        //    width: '100%',
+        //},
+        layout: 'hbox',
+        items: [collapseButton, expandButton],
```

Comment:
There was no need to create vars for the two buttons, you could have left the items definition unchanged. But if you are doing it for consistency with the rest of the code, that's fine.

Reply:
Indeed, that's the main reason. For me it is easier to read this way.